lib/auth/usertoken.go

```diff
@@ -382,6 +403,7 @@ func formatUserTokenURL(proxyHost string, tokenID string, reqType string) (strin
 		u.Path = fmt.Sprintf("/web/recovery/steps/%v/verify", tokenID)
 	}
 
+	// TODO: case for UserTokenTypeBot?
```

Comment:
That's probably the best way forward if we continue using user tokens. That said, if we switch to regular provisioning tokens  it might be a moot point.